github.user.styl

```diff
@@ -950,38 +952,64 @@ for c, i in C {
 
         &[type='radio'],
         &[type='checkbox'] {
-            &, &:focus, &:hover:active, &:disabled, &:hover:active:disabled {
+            &, &:focus, &:hover:active {
                 -webkit-appearance: none
                 -moz-appearance: none
                 appearance: none
                 cursor: pointer
-                height: 1rem
-                width: 1rem
+                height: b_hgwd
+                width: b_hgwd
                 border: 1px solid bg5
                 background-size: cover
                 c: fg1 0 bg1
             }
+            &:disabled, &:hover:active:disabled {
+                c: fg1 0 bg4
+            }
         }
 
-        &[type='radio']:checked {
-            z = '<circle cx="256" cy="256" r="128" fill="white"/>'
-            background-image: s("url('%s%s</svg>') %s", r(prefix), r(z), i)
-            border-radius: 100% i
-            &:checked { c: 0 ac4 ac4 }
+        &[type='radio'] {
+            /// fix border radius at https://github.com/settings/profile Emoji color
+            border-radius: 50%
+            &:checked {
+                z = '<circle cx="256" cy="256" r="128" fill="white"/>'
+                background-image: s("url('%s%s</svg>') %s", r(prefix), r(z), i)
+                c: 0 ac2 bg2
```

Comment:
Not a fan of these colors, they hardly stand out.

Reply:
before `{ c: 0 ac4 ac4 }` background cant be accent color and color is brighter than before

Reply:
I'm not sure what you're trying to say.

Reply:
These changes are about checkbox states and differentiation between disabled and enabled. 

The default background is open to interpretation so dealers choice, if you dislike something feel free to use `ctrl +g` to submit a suggestion

Reply:
I have no clue what _your_ default styles look like. Mine look like this:

![image](https://user-images.githubusercontent.com/18245694/100436881-40809000-30a0-11eb-865c-05c525fa8c35.png)

Reply:
Linux vs WIndows?

Reply:
I don't think there's a way to do that. Perhaps it would look better if you'd use only background colors. Let me do the rebase voodoo for this branch so that you can push your changes.

Reply:
dont do any voodo and break what Im working on a rebased brach already

Reply:
Done! What do you think of these colors?

![image](https://user-images.githubusercontent.com/18245694/100449252-e25da800-30b3-11eb-861f-cdb2afb77e0f.png)